content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -895,10 +895,10 @@ The `Unsupported` type was introduced in [2.17.0](https://github.com/prisma/pris
 
 ```prisma
 model Star {
-  id           Int                      @id @default(autoincrement())
-  position     Unsupported("circle")?
-  example1     Unsupported("circle")
-  circle       Unsupported("circle")?  @default(dbgenerated("'<(10,4),11>'::circle"))
+id           Int                      @id @default(autoincrement())
+position     Unsupported("circle")?
+example1     Unsupported("circle")
+circle       Unsupported("circle")?  @default(dbgenerated("'<(10,4),11>'::circle"))
```

Comment:
This is wrongly formatted right now, probably because it is missing the closing bracket:
```suggestion
circle       Unsupported("circle")?  @default(dbgenerated("'<(10,4),11>'::circle"))
}
```